tests/cli/test_ingest.py

```diff
@@ -117,54 +120,79 @@ def test_dbt_flush_test_results_soda_server_scan_test_result(
     assert test_results[0][column] == value
 
 
-@pytest.mark.parametrize(
-    "dbt_artifacts, dbt_manifest, dbt_run_results, expectation",
-    [
-        pytest.param(
-            Path('my_dbt_project/target/'),
-            "",
-            "",
-            [
-                Path('my_dbt_project/target/manifest.json'),
-                Path('my_dbt_project/target/run_results.json'),
-            ],
-            id="dbt_artifacts path provided, others null"
-        ),
-        pytest.param(
-            Path('my_dbt_project/target/'),
-            Path('my_dbt_project/path_to_ignore'),
-            Path('my_dbt_project/path_to_ignore'),
-            [
-                Path('my_dbt_project/target/manifest.json'),
-                Path('my_dbt_project/target/run_results.json'),
-            ],
-            id="dbt_artifacts path provided, others provided, but should be ignored"
-        ),
-    ]
-)
-def test_resolve_artifacts_paths(dbt_artifacts, dbt_manifest, dbt_run_results, expectation):
-    dbt_manifest, dbt_run_results = resolve_artifacts_paths(dbt_artifacts, dbt_manifest, dbt_run_results)
-    assert dbt_manifest == expectation[0]
-    assert dbt_run_results == expectation[1]
-
-
 @pytest.mark.parametrize(
     "dbt_artifacts, dbt_manifest, dbt_run_results",
     [
         pytest.param(
-            "",
-            "",
-            Path('my_dbt_project/target/run_results.json'),
+            None,
+            None,
+            Path("my_dbt_project/target/run_results.json"),
             id="missing dbt_manifest and artifact",
         ),
         pytest.param(
-            "",
-            Path('my_dbt_project/target/run_results.json'),
-            "",
+            None,
+            Path("my_dbt_project/target/manifest.json"),
+            None,
             id="missing dbt_run_results and artifact",
         ),
-    ]
+    ],
```

Comment:
shouldn't we add a test case for the scenario where we provide a `dbt_artifacts` only?

Reply:
done!

Reply:
Are you sure you pushed your change? I don't see the test case I'm expecting in the file